tests/test_src/test_transaction/test_transaction.cpp

```diff
@@ -703,3 +703,64 @@ TEST_P(PrepareEsdtTransferData, noFunction)
     EXPECT_EQ(tx.m_gasLimit, currParam.gasLimitAfterPrep);
     EXPECT_EQ(txDataAfterPrep, currParam.dataAfterPrep);
 }
+
+TEST(ESDTProperties, comparisonOperators)
```

Comment:
Perhaps in the future we might want to extract these ESDT-specific tests to a separate file.

Reply:
Yes. It would be worth splitting them in the future as more tests are going to be added.